AlfrescoApp/Model/Managers/RealmSyncManager.m

```diff
@@ -1000,4 +1071,97 @@ - (NSString *)accountIdentifierForAccount:(UserAccount *)userAccount
     return accountIdentifier;
 }
 
+/*
+ * shows if sync is enabled based on cellular / wifi preference
+ */
+- (BOOL)isSyncEnabled
+{
+    UserAccount *selectedAccount = [[AccountManager sharedManager] selectedAccount];
+    BOOL syncPreferenceEnabled = selectedAccount.isSyncOn;
+    BOOL syncOnCellularEnabled = [[PreferenceManager sharedManager] shouldSyncOnCellular];
+    
+    if (syncPreferenceEnabled)
+    {
+        BOOL isCurrentlyOnCellular = [[ConnectivityManager sharedManager] isOnCellular];
+        BOOL isCurrentlyOnWifi = [[ConnectivityManager sharedManager] isOnWifi];
+        
+        // if the device is on cellular and "sync on cellular" is set OR the device is on wifi, return YES
+        if ((isCurrentlyOnCellular && syncOnCellularEnabled) || isCurrentlyOnWifi)
+        {
+            return YES;
+        }
+    }
+    return NO;
+}
+
+- (void)showSyncAlertWithCompletionBlock:(void (^)(BOOL completed))completionBlock
```

Comment:
This method is not needed anymore as favorites is no longer connected with sync.